src/api/app/components/bs_request_overview_avatars_component.rb

```diff
@@ -40,4 +36,15 @@ def avatars_to_display
   def number_of_hidden_avatars
     [0, avatar_objects.size - MAXIMUM_DISPLAYED_AVATARS].max
   end
+
+  def package_avatar_objects
+    relationships = @review.package&.relationships.for_maintainer_and_reviewer_roles.includes(:user, :group)
+    reviewers = relationships.map { |relation | relation.user_id.present? ? relation.user : relation.group.users }.flatten.uniq!
```

Comment:
This should be pushed to the `Review` model, we should tell `Review` to give us it's reviewers instead of asking too many details from it to do it by ourselves.

Reply:
Also, I'm not sure we want to use`uniq!` (see [the api doc](https://rubydoc.info/stdlib/core/3.0.0/Array:uniq!)) as it returns `nil` if no duplicates are found instead returning the original array...